packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -85,7 +85,11 @@ export const DevServerMixin = {
      * @private
      */
     _setupLogging(app) {
-        app.use(expressLogging('dev'))
+        app.use(
+            expressLogging(
+                '(:req[correlation-id]) :method :url :status :response-time ms - :res[content-length]'
+            )
+        )
```

Comment:
Alex asked "will this break anything?". I'd say nope!

I'm pretty sure you want to configure logging for production though, too. I'm not even sure how we do that right now.